src/core/createOptIn.js

```diff
@@ -47,12 +47,17 @@ export default () => {
     /**
      * Only to be called by the Privacy component during startup. If opt-in
      * isn't enabled, this method will not be called.
-     * @param {Object} _cookie The cookie management object, namespaced
+     * @param {Object} logger A logger object.
+     * @param {Object} _cookie A cookie management object.
      * to the Privacy component.
      */
-    enable(_cookie) {
+    enable(logger, _cookie) {
       cookie = _cookie;
       purposes = cookie.get(COOKIE_NAMESPACE) || PENDING;
+
+      if (purposes === PENDING) {
```

Comment:
I would extract the message into constant, so you could reference it in different places.

Reply:
@artur-ciocanu Are you just talking about putting `"Some commands may be delayed until the user opts in."` as a constant at the top of the file, or are you saying expose it in such a way that other modules or the test can also reference it? Are you suggesting we put all log messages in constants?